app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
It happens @himdel. In our tests, once that still had not a property named "vendor" in Factory which is used to make seed, the tests throws errors exactly in the `downcase` call.

I had a manual "test" in execution: I update the vendor name to nil and save it, after that, a 500 error appeared in ManageIQ interface because of `downcase` call.